new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -38,6 +44,37 @@ public static String createUniqueNamespace() {
         return Namespace.createUniqueNamespace();
     }
 
-    // etc...
+    // ------------------------ Docker image  -------------------------
+
+    public static boolean createMIIImage(
+        String baseImageName, 
+        String imageTag,
+        List<String> modelFiles,
+        List<String> modelVariableFiles,
+        List<String> modelArchiveFiles,
+        String wdtVersion,
+        String domainType) {
+  
+    	return 
+            new WebLogicImageTool()
+                .with(new WITParams()
+                     .baseName(baseImageName).tag(imageTag).modelFiles(modelFiles)
+                     .modelVariableFiles(modelVariableFiles).modelArchiveFiles(modelArchiveFiles)
+                     .wdtVersion(wdtVersion).domainType(domainType)).updateImage();
+    }
+ 
+ 
+    // ------------------------ Installer  -------------------------
+
+    public static boolean verifyAndInstallWIT(String version, String location) {
```

Comment:
IMO, ideally version/location would itself come from a global type - the net idea is that at the beginning of a suite, all such settings are defined once (and only once) and then passed around.  Like the 'packet' object in the operator code.

Reply:
I hope we only need to do download/install of WIT and WDT binary once for the entire integration test suite. We could make them static constants, but parameterizing them makes it a little easier for the test suite to test a newer version (when the time comes).

Reply:
i think it makes sense to download all of the prereqs up front (in the jenkins job) before we start running the tests - e.g. WIT, WDT, whatever installer/patches we need, helm, kubectl, etc.

Reply:
Let's put the prerequisite downloads in the Jenkins file.  I think it will be much easier to maintain.  And, if we switch to another environment like running in Docker containers with the software pre-positioned, the change is very minor.

Reply:
Agreed. We'll need to have a way for the test code to get the location of WIT and WDT binary; the MII use case needs to run WIT to generate a nee image that contains the WDT binary.

Reply:
Its easy for the Jenkinsfile to download the bits, extract them, and then put the folder/path in an environment variable.  Will that work?

Reply:
@ddsharpe I think we shjould avoid depending on Jenkins for any critical part of the flow.  It should be possible to get a test up and running, _with a single mvn call_, in any basic WKO env. We shouldn't need to install Jenkins to get it working IMO - unless installing Jenkins and setting it up is itself one step?

Reply:
So would relying on a Jenkinsfile also rely on Jenkins being installed?

Reply:
Derek and I have agreed to disagree off-line - and are looking for other opinions.  

Minimally I'd like to see (1) single or at most two places that 'own' setting up WDT/WIT, (2) target/scope install location of WDT/WIT to the the test suite (cleaned up when test suite is deleted - saved if along with test results if test suite results are saved), (3) it must be very easy to get a test running stand-alone (if missing the test infra should get it - or at least a single command for setting up prior to test suite).

Reply:
If we offload the WIT/WDT download  responsibly to Jenkin Script, then to run the test locally we need to add  a similar script to the operator github project. There may be out-of-sync  issue between  Jenkin and a local copy